vault/resource_pki_secret_backend_role.go

```diff
@@ -297,6 +297,14 @@ func pkiSecretBackendRoleResource() *schema.Resource {
 				Description: "Flag to mark basic constraints valid when issuing non-CA certificates.",
 				Default:     false,
 			},
+			"not_before_duration": {
+				Type:         schema.TypeString,
+				Required:     false,
+				Optional:     true,
+				Description:  "Specifies the duration by which to backdate the NotBefore property.",
+				ValidateFunc: validateDuration,
+				Default:      "30s",
```

Comment:
Could we avoid setting a default here so Vault's default takes over? That way if Vault ever changes the default to something else, it'll get picked up right away.

Reply:
I now leave it to Vault to set a default.  FYI, many other settings on this role define a default  instead of letting Vault do it.

Reply:
@tyrannosaurus-becks -- need anything else from me?

Reply:
Nope, perfect. Thanks!